e2e-tests/gef/cypress/e2e/facility/pre-submission/facility-end-date-disabled/facilities-change-from-details-journey.spec.js

```diff
@@ -0,0 +1,84 @@
+import relative from '../../../relativeURL';
+import applicationDetails from '../../../pages/application-details';
+import facilityGuarantee from '../../../pages/facility-guarantee';
+import { BANK1_MAKER1 } from '../../../../../../e2e-fixtures/portal-users.fixture';
+
+const applications = [];
+let token;
+
+context('Changing facility details from application-details page should take you to next page on facilities journey', () => {
```

Comment:
For all FF E2E tests, can we use the convention we made in [this PR](https://github.com/UK-Export-Finance/dtfs2/pull/3568/files#diff-2269db3a52a9d7f21a02f7ff3b94adea82f4a3a00b281e2787d49239ddeb12a1R7) please, "X flag disabled/enabled"?